generators/filesystem/main.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+import sys
+import os
+import csv
+import fnmatch
+import datetime
+from time import strftime
+from unicode_csv import UnicodeDictReader
```

Comment:
Remove this import and the file. If you choose to copy files from other generators, make sure to evaluate **everything** so there is no unnecessary code.